diracos/scriptTemplates/set_RPATH.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
```

Comment:
We now have a mix of python2.6, mock, python2.7 and python3... the ECGD groups should learn from our diversity...

Reply:
If you want find and add a up-to-date `patchelf` and `lief` SPRMs be my guest :wink:

Reply:
No no, I knew this was the reason, just making the point :D

Reply:
This is indent 4 python, please format the default way